_episodes/01-numpy.md

```diff
@@ -1135,3 +1135,73 @@ the graphs will actually be squeezed together more closely.)
 > > {: .output}
 > {: .solution}
 {: .challenge}
+
+>## Change In Inflamation
+>
+>This patient data is _longitudinal_ in the sense that each row represents a
+>series of observations relating to one individual. This means that change
+>inflamation is a meaningful concept.
+>
+>The `numpy.diff()` function takes a NumPy array and returns the 
+>difference along a specified axis.
+>
+>Which axis would it make sense to use this function along?
+> > ## Solution
+> > Since the row axis (0) is patients, it does not make sense to get the
+> > difference between two arbitrary patients. The column axis (1) is in
+> > days, so the differnce is the change in inflamation -- a meaningful
+> > concept.
+> >
+> > ~~~
+> > numpy.diff(data, axis=1)
+> > ~~~
+> > {: .python}
+> {: .solution}
+>
+>If the shape of an individual data file is `(60, 40)` (60 rows and 40 columns)
+>, what would the shape of the array be after you run the `diff()` function and
```

Comment:
Could you put the comma on the previous line? Better to break the parenthetical statement, or move that statement to the next line, than to have a space before the comma.